components/slider/index.css

```
@@ -244,36 +265,43 @@ governing permissions and limitations under the License.
     /* Flip the ramp automatically for RTL */
     transform: logical rotate(0deg);
   }
+
+  &+.spectrum-Slider-handle {
+    margin-block: calc(var(--mod-slider-bottom-to-handle, var(--spectrum-slider-bottom-to-handle)) * -1 -5px + var(--mod-slider-margin-top-offset, var(--spectrum-slider-margin-top-offset))) 0;
```

```suggestion
    margin-block: calc(
    var(--mod-slider-bottom-to-handle, var(--spectrum-slider-bottom-to-handle)) * -1 - 5px + var(--mod-slider-margin-top-offset, var(--spectrum-slider-margin-top-offset))) 0;
```

I noticed the missing space between the subtraction & 5px was causing the calc to render without it.

---

good catch! thanks for seeing this.